pkg/virt-api/rest/usbredir.go

```diff
@@ -0,0 +1,34 @@
+package rest
+
+import (
+	"fmt"
+
+	restful "github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	apimetrics "kubevirt.io/kubevirt/pkg/monitoring/api"
+)
+
+func (app *SubresourceAPIApp) USBRedirRequestHandler(request *restful.Request, response *restful.Response) {
+	activeConnectionMetric := apimetrics.NewActiveVNCConnection(request.PathParameter("namespace"), request.PathParameter("name"))
```

Comment:
A small oversight here: We will need our own metric for  that. This would reuse the metric for VNC.

Reply:
Ah, indeed. I did not pay enough attention while addressing the last rebase. Thanks!